styles.css

```diff
@@ -134,8 +134,8 @@ hr {
 }
 
 #img-project {
-  width: 20.438rem;
-  height: 14.813rem;
+  /* width: 20.438rem;
+  height: 14.813rem; */
```

Comment:
- [OPTIONAL] You have done all that was required which is the reason why I am making this an optional suggestion. Kindly remove the commented code from this file. It is a good practice to make your code look clean and commented codes like this make the code look untidy. 👍 💻